### ISSUE TITLE
403 Error Access denied while downloading pretrained models

### ISSUE THREAD

**Issue by owoshch** [mit-han-lab/spvnas#68]
Hi! Thank you for your wonderful work!

I'm trying to reproduce your results on validation set using the pretrained weights you provided.

Unfortunately, I'm getting 403 Error "Access forbidden! You don't have permission to access the requested directory." when following every link for pretrained models, for example: https://hanlab.mit.edu/files/SPVNAS/spvnas_specialized/SemanticKITTI_val_SPVNAS@65GMACs/

Is there any way to fix this issue?

Thank you.

**Comment by CCInc** [comment]
@owoshch it looks like something got broken, could you take a look @zhijian-liu ?

**Comment by owoshch** [comment]
Thank you for taking this issue into consideration! 
I was wondering what is the suggested way to use the pretrained models.
For example, I was able to load model from the code like this `model = spvnas_specialized('SemanticKITTI_val_SPVNAS@65GMACs').cuda()` but was unable to access the link to the model from the browser

**Comment by CCInc** [comment]
The direct links to the models still work, it's just the folder display that is broken. So, I would suggest using the model zoo for now.

**Comment by zhijian-liu** [comment]
It seems that our server has updated the folder permission. Nevertheless, you can load and use the model from the model zoo as @CCInc suggested.
